tests/net_without_seccomp.rs

```diff
@@ -0,0 +1,59 @@
+#[cfg(target_os = "linux")]
+use std::collections::BTreeMap;
+#[cfg(target_os = "linux")]
+use std::net::TcpStream;
+
+#[cfg(target_os = "linux")]
+use birdcage::{Birdcage, Sandbox};
+#[cfg(target_os = "linux")]
+use seccompiler::{
+    BpfProgram, SeccompAction, SeccompCmpArgLen, SeccompCmpOp, SeccompCondition, SeccompFilter,
+    SeccompRule, TargetArch,
+};
+
+#[cfg(target_os = "linux")]
+#[cfg(target_arch = "x86_64")]
+const ARCH: TargetArch = TargetArch::x86_64;
+#[cfg(target_os = "linux")]
+#[cfg(target_arch = "aarch64")]
+const ARCH: TargetArch = TargetArch::aarch64;
+
+#[cfg(target_os = "linux")]
+fn main() {
+    // Create seccomp filter blocking seccomp prctl syscall.
+    let mut rules = BTreeMap::new();
+    let seccomp_prctl = SeccompCondition::new(
+        0,
+        SeccompCmpArgLen::Dword,
+        SeccompCmpOp::Eq,
+        libc::PR_SET_SECCOMP as u64,
+    )
+    .unwrap();
+    let rule = SeccompRule::new(vec![seccomp_prctl]).unwrap();
+    rules.insert(libc::SYS_prctl, vec![rule]);
```

Comment:
It might make sense to block the `seccomp` system call as well just in case the implementation in `seccompiler` changes in the future.

```suggestion
    rules.insert(libc::SYS_prctl, vec![rule]);
    rules.insert(libc::SYS_seccomp, Vec::new());
```

Reply:
Couldn't hurt.